Signal/src/call/PeerConnectionClient.swift

```diff
@@ -128,17 +144,23 @@ class PeerConnectionClient: NSObject, RTCPeerConnectionDelegate, RTCDataChannelD
     // MARK: - Media Streams
 
     public func createSignalingDataChannel() {
-        let dataChannel = peerConnection.dataChannel(forLabel: Identifiers.dataChannelSignaling.rawValue,
-                                                     configuration: RTCDataChannelConfiguration())
-        dataChannel.delegate = self
+        AssertIsOnMainThread()
+
+        PeerConnectionClient.signalingQueue.sync {
```

Comment:
Perhaps PeerConnectionClient's constructor should just call createSignalingDataChannel().
We always do it immediately after instantiating the PCC.  Anyway, no need for a promise, right?